tests/MonologCreator/FactoryTest.php

```diff
@@ -83,6 +83,8 @@ public function testCreateDefaultLoggerFail($configString)
     {
         $config = json_decode($configString, true);
 
+        $this->expectException(\MonologCreator\Exception::class);
```

Comment:
You forgot to remove the `@expectedException` annotation.

Reply:
right and fixed.